openvino_tensorflow/encapsulate_clusters.cc

```diff
@@ -289,6 +289,19 @@ Status Encapsulator::AnalysisPass() {
             false, &((*(new_input_node_def->mutable_attr()))["_is_variable"]));
       }
 
+      if (dst->type_string() == "SparseToDense" && edge->dst_input() == 1) {
```

Comment:
Can we add comments here stating why this temp change is required, and for what class of models?